Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -1195,6 +1195,7 @@
         SYS_bsdthread_terminate
         SYS_change_fdguard_np
         SYS_close_nocancel
+        SYS_crossarch_trap
```

Comment:
I don’t think this is required. Should we just silence?

Reply:
You do not think it is needed in the GPU process? Or in any process at all?